go.mod

```diff
@@ -1,6 +1,7 @@
 module sigs.k8s.io/cluster-api-provider-vsphere
 
 go 1.22.0
+toolchain go1.22.8
```

Comment:
Wow :/

(cc @chrischdi that's not good)

Reply:
No idea if that's correlated to go 1.22.7 in here: https://github.com/kubernetes-sigs/kustomize/blob/kyaml/v0.18.0/kyaml/go.mod